middleware/logger.go

```diff
@@ -34,7 +64,7 @@ func Logger() echo.MiddlewareFunc {
 				code = color.Cyan(n)
 			}
 
-			log.Printf("%s %s %s %s %d", method, path, code, stop.Sub(start), size)
+			log.Printf("%s %s %s %s %s %d", parseIP(remoteAddr), method, path, code, stop.Sub(start), size)
```

Comment:
I understand, getting real IP should be the case. Now, in `parseIP` are we separating host and port and then calling `net.ParseIP`? If yes, do you think we can use http://golang.org/pkg/net/#SplitHostPort?